tests/unit/functions/github.test.ts

```
@@ -49,26 +63,32 @@ describe("createGithubTeam", () => {
-        { name: "Test Team", id: existingTeamId },
-      ],
-    });
+    mockOctokit.paginate.iterator.mockReturnValue(
+      createAsyncIterator([
+        {
+          data: [
+            { name: "Other Team", id: 789 },
+            { name: "Test Team", id: existingTeamId },
+          ],
+        },
+      ])
```

ESLint

[error] Insert `,`

(prettier/prettier)